src/ixdat/backends/directory_backend.py

```diff
@@ -4,11 +4,29 @@
 from ..config import CFG
 
 
+char_substitutions = {
+    "/": "_DIV_",  # slash (divided by)
+    "\\": "_BKSL_",  # backslash
+    ".": "_DOT_",  # decimal
+    "^": "_CFLX_",  # circumflex accent (raised-to-the)
+    "<": "_LTS_",  # less-than sign
+    ">": "_GTS_",  # greater-than sign
+}
+
+
+def fix_name_for_saving(name):
```

Comment:
Good idea. Problems like these are annoying, because ultimately it can be difficult to make sure you account for everything. If it turns out to be a problem in the future, you can also take an approach where after replacing the most obvious, you simple try and create the file in a tmp dir (https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) and see if it succeeds. Then if it fails, replace the offending characters with blank spaces or the Unicode code point name.

Reply:
This is enough for all BioLogic columns so far. TODO left with link, for when we need the solution to be guaranteed for new evil datasets.